### PR TITLE
Admin Guide: Virtualization Rewrite

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8255

### CODE REVIEW

File: modules/client-configuration/pages/virt-file.adoc [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
-File-based is suitable for VMWare vCenter installations for which no direct API access is possible from the {susemgr}.
+. Export the JSON configuration file by running [command]``virtual-host-gatherer`` on the VM network.
+. Save the produced file to a location accessible by your {productname} Server.
+. In the {productname} {webui}, navigate to menu:Systems[Virtual Host Managers].
+. Click btn:[Create] and select [guimenu]``File-based``.
+. In the [guimenu]``Add a file-based Virtual Host Manager`` section, use these parameters:
+* In the [guimenu]``Label`` field, type a custom name for your VHM.
+* In the [guimenu]``Url`` field, type the path to your exported JSON configuration file.
+. Click btn:[Create] to save your changes and create the VHM.
+. On the [guimenu]``Virtual Host Managers`` page select the new VHM.
```
Comment (keichwa) [inline modules/client-configuration/pages/virt-file.adoc]:
```suggestion
. On the [guimenu]``Virtual Host Managers`` page, select the new VHM.
```

File: modules/client-configuration/pages/virt-vmware.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+[[virt-vmware]]
+= Virtualization with VMWare
+
+You can use VMWare vSphere virtual machines, including ESXi and vCenter, with {productname} by setting up a virtual host manager, or VHM.
```
Comment (keichwa) [inline modules/client-configuration/pages/virt-vmware.adoc]:
```suggestion
You can use VMWare vSphere virtual machines, including ESXi and vCenter, with {productname} by setting up a virtual host manager (VHM).
```

File: modules/client-configuration/pages/virt-vmware.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+.Procedure: Creating a VMWare VHM
+
+. In the {productname} {webui}, navigate to menu:Systems[Virtual Host Managers].
+. Click btn:[Create] and select [guimenu]``VMWare-based``.
+. In the [guimenu]``Add a VMWare-based Virtual Host Manager`` section, use these parameters:
+* In the [guimenu]``Label`` field, type a custom name for your VHM.
+* In the [guimenu]``Hostname`` field, type the fully-qualified domain name (FQDN) or host IP address.
+* In the [guimenu]``Port`` field, type the ESXi API port to use (for example, [parameter]``443``.)
```
Comment (keichwa) [inline modules/client-configuration/pages/virt-vmware.adoc]:
```suggestion
* In the [guimenu]``Port`` field, type the ESXi API port to use (for example, [parameter]``443``).
```
I think this style is used elsewhere.

File: modules/client-configuration/pages/virtualization.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+= Virtualization
+
+You can use {productname} to manage virtualized clients in addition to regular traditional or Salt clients.
+In this type of installation, a virtual host is installed on the {productname} Server to manage any number of virtual guests.
+If you choose to, you can install several virtual hosts to manage groups of guests.
+
+The range of capabilities that virtualized clients have depends on the third-party virtualization provider you choose.
+
+Xen and KVM hosts and guests can be managed directly in {productname}.
+This enables you to autoinstall hosts and guests using {ay} or {kickstart}, and manage guests in the {webui}}.
```
Comment (keichwa) [inline modules/client-configuration/pages/virtualization.adoc]:
```suggestion
This enables you to autoinstall hosts and guests using {ay} or {kickstart}, and manage guests in the {webui}.
```

File: modules/client-configuration/pages/virtualization.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+For VMWare, including VMWare vSphere, {productname} requires you to set up a virtual host manager, or VHM, to control the VMs.
```
Comment (keichwa) [inline modules/client-configuration/pages/virtualization.adoc]:
```suggestion
For VMWare, including VMWare vSphere, {productname} requires you to set up a virtual host manager (VHM) to control the VMs.
```

**Comment by Loquacity** [comment]
I'm going to merge this since no one is yelling at me. I can't cope with any more merge conflicts :\

**Pull request merged**